example/timer/odp_timer_accuracy.c

```diff
@@ -691,13 +692,22 @@ int main(int argc, char *argv[])
 	}
 
 	if (test_global.opt.output) {
-		test_global.log = calloc(test_global.tot_timers,
-					 sizeof(test_log_t));
+		odp_shm_t shm;
+		void *addr;
+		uint64_t size = test_global.tot_timers * sizeof(test_log_t);
 
-		if (test_global.log == NULL) {
-			printf("Test log calloc failed.\n");
+		shm = odp_shm_reserve("timer_accuracy_log", size,
+				      sizeof(test_log_t), 0);
+
+		if (shm == ODP_SHM_INVALID) {
+			printf("Test log alloc failed.\n");
 			goto quit;
 		}
+
+		addr = odp_shm_addr(shm);
```

Comment:
Since this is an example application, I would still check the return value here.

Reply:
Shm handle is checked before the call, and memset(0) after the call crashes if some address of the memory block  cannot be written. So, I didn't think that addr != NULL check would have a benefit here.

Reply:
Yes, the check is rather unnecessary here. I just had a faint memory that we previously changed all example applications to check odp_shm_addr() return value, since according to api it may fail. I'm good either way.

Reply:
I think those apps didn't check either shm handle or pointer. I think it's enough to check handle or pointer, but no need to check both.

Reply:
OK